README.md

```diff
@@ -59,33 +59,38 @@ python -m scoop -n N -vvv generate_template.py
 
 ## How to generate your own template?
```

Comment:
update to reflect what are the steps:
- edit configuration.json
- run pipeline.py (which calls preprocessing.py)
- generate_template.py

Reply:
i find it confusing to have "pipeline" as the preproc. Consider having a file called "run_preprocessing" for running preprocessing.

consider getting rid of pipeline, or merging with something else, etc

Reply:
encapsulation of scripts: we need a clean organizatoin of scripts vs. API vs. child scripts

ask other soft engeer in the lab for help